frontend/src/components/explorer/mapViewer/MapControls.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="canvasOption overlay">
+  <div class="canvasOption overlay p-">
```

Comment:
Looks like it's missing a number here, maybe `p-2`?

Reply:
Absolutely!